app/app.js

```diff
@@ -6,6 +6,7 @@ import config from './config/environment';
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
 var App = Ember.Application.extend({
+  LOG_TRANSITIONS: true,
```

Comment:
I know this branch is in progress, but just so I've done due diligence: remove logging before merging this into master.